src/test/scala/uk/gov/nationalarchives/tdr/api/service/FileServiceSpec.scala

```diff
@@ -39,9 +40,14 @@ class FileServiceSpec extends AnyFlatSpec with MockitoSugar with Matchers with S
     when(consignmentRepositoryMock.addParentFolder(consignmentId, "Parent folder name")).thenReturn(mockConsignmentResponse)
     val mockFileMetadataResponse = Future.successful(Seq(FilemetadataRow(UUID.randomUUID(), fileId, "value", Timestamp.from(Instant.now), uuid, "name")))
     when(fileMetadataRepositoryMock.addFileMetadata(any[Seq[FilemetadataRow]])).thenReturn(mockFileMetadataResponse)
+
     val fileMetadataService = new FileMetadataService(fileMetadataRepositoryMock, FixedTimeSource, fixedUuidSource)
     val ffidMetadataService = new FFIDMetadataService(ffidMetadataRepositoryMock, mock[FFIDMetadataMatchesRepository], FixedTimeSource, fixedUuidSource)
-    val fileService = new FileService(fileRepositoryMock, consignmentRepositoryMock, fileMetadataService, ffidMetadataService, FixedTimeSource, fixedUuidSource)
+    val antivirusMetadataService = new AntivirusMetadataService(mock[AntivirusMetadataRepository])
```

Comment:
Is it worth placing the `mock[AntivirusMetadataRespository]` as a class val as per the mock repositories, as this is repeated a lot in the class?

Reply:
Yeah, you're right. I've put a val at the top of the class.